.github/workflows/build.yml

```diff
@@ -3,20 +3,20 @@ on:
   pull_request:
```

Comment:
What if we just ran the linter on the folder `cluster/examples/kubernetes`? The github actions will self-validate by failing the CI if there is a problem. But I guess if it makes syntax errors more obvious in these files, it would be good to keep.